src/aap_eda/core/migrations/0051_auto_20241011_1449.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.7 on 2024-10-11 14:49
```

Comment:
This migration is not generated by django.

Reply:
I generated it using `makemigrations core --empty`